pom.xml

```diff
@@ -30,7 +30,7 @@
     <version>5.5-SNAPSHOT</version>
   </parent>
   <artifactId>xwiki-plugin-scheduler</artifactId>
```

Comment:
if we fork it, can we drop the xwiki artifact/project naming? wdyt about e.g. `celements-scheduler`?
also we've so far added xwiki forks to the `celements-xwiki` git repo. is there a specific reason why this is in it's own repository?

Reply:
I kept to the xwiki-lucene-plugin notation. Putting everything in one repository will get awkward complicated and we end with a very havy repository. The idea of the celements-xwiki repository IMHOP is, that the xwiki-celements abstraction layer resides therein. Everything which abstracts from the xwiki layer and should not be directly accessed by the celements programmings.
Suggestion: I could start a celements-plugin / celements-module directory where we can move all the celements-plugins and modules. This would be a celements-lucene, celements-scheduler but too the celements-blog, celements-calender, celements-image
wdyt?

Reply:
Yes, I understand, Sounds good to me! You could directly rename this repo and move the existing files to a `celements-scheduler` subdirectory.

Reply:
done.